spec/lib/object_serializer_notifications_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+describe FastJsonapi::ObjectSerializer do
+  include_context 'movie class'
+
+  context 'instrument' do
```

Comment:
it would be more readable if the spec were named a little elaborately. It's important to test both cases.
```
context 'when calling serializable_hash' do
  it 'should send a notification' do
  end
  
  it  'shouldnt send a notification' do
  end
end
```

Reply:
Of course. These tests were mostly just to make sure that I was actually getting notifications. :p 

TDD FTW!!!